UI/DesignWnd.cpp

```diff
@@ -2736,7 +2736,7 @@ void BasesListBox::ResetEmptyListPrompt()
 
 void CompletedDesignsListBox::ResetEmptyListPrompt() {
     if (!GetOptionsDB().Get<bool>("resource.shipdesign.saved.enabled")
-        && !GetOptionsDB().Get<bool>("resource.shipdesign.default.enabled"))
+        /*&& !GetOptionsDB().Get<bool>("resource.shipdesign.default.enabled")*/)
```

Comment:
I think this change isn't quite right.  In all the the other places you are explicitly or implicitly replacing ```GetOptionsDB().Get<bool>("resource.shipdesign.default.enabled")``` with ```true``` whereas  here it looks like you may have overlooked the "!", since just commenting out this line winds up implicitly treating the logical clause like it was true, meaning treating the option like it was false.

I think the most textually simple change to make here, in keeping with the changes you made elsewhere, would be to not comment out the whole line, but change it to
```
        && false /*!GetOptionsDB().Get<bool>("resource.shipdesign.default.enabled")*/)
```

Reply:
If I recall correctly, I tried to male things as though the extra option didn't exist. If you make it `&& false` the prompt will never display.

Reply:
>  If you make it && false the prompt will never display.

Correct; the prompt doesn't really make sense if the option is not available.   A more complete removal would be to remove the whole section of the if statement, but it seems your goal with the other changes was just to disable rather than completely remove, and simple disablement seems fine to me.  But in this particular case I think that means not showing this prompt at all.


```
NO_SAVED_OR_DEFAULT_DESIGNS_ADDED_PROMPT
'''Select a hull, add parts and then click "[[DESIGN_WND_ADD_FINISHED]]" to create an available design.

Neither the Saved Designs nor the Default Designs were added to the empire at game start.  Options to add these designs at game start can be changed in Options->Other.  Add the Saved Designs immediately with the context menu of any design under the Saved tab.'''
```

Reply:
as you prefer...

Reply:
> Correct; the prompt doesn't really make sense if the option is not available.

There is still the option for the saved designs in this case though...